plugin-server/src/cdp/hog-executor.ts

```diff
@@ -106,15 +109,30 @@ export class HogExecutor {
             if (hogFunction.filters?.bytecode) {
                 const start = performance.now()
                 try {
-                    const filterResult = execHog(hogFunction.filters.bytecode, { globals: filtersGlobals })
+                    const filterResult = execHog(hogFunction.filters.bytecode, {
+                        globals: filtersGlobals,
+                        telemetry: true,
```

Comment:
Feels like we should already use an env var for this tbh. I also like the lru option for turning it on for dodgy fields but if this does cause perf issues beyond what we expected it would be nice to be able to just switch it off easily. There is a super easy helper `buildIntegerMatcher(hub.CDP_HOG_EXCUTOR_TELEMETRY_FUNCTIONS)` for example so that we can enable it conditionally

Reply:
Sweet, this does make it easy. I added an env (default to `''`, supports `'*'`) and a test to make sure it logs the telemetry.